static/styles/left-sidebar.scss

```diff
@@ -507,6 +508,7 @@ li.expanded_private_message a {
 #streams_header {
     margin-right: 0px;
     padding-left: $far_left_gutter_size;
+    padding-right: 10px;
```

Comment:
This does not match your screenshot.

Reply:
ahh... I missed the private message scrollbar

Reply:
I removed this css line, felt like it's not actually not required